packages/react/src/components/TextInput/PasswordInput.tsx

```diff
@@ -329,18 +349,18 @@ const PasswordInput = React.forwardRef(function PasswordInput(
         data-toggle-password-visibility={inputType === 'password'}
       />
       {isFluid && <hr className={`${prefix}--text-input__divider`} />}
-      <button
-        type="button"
-        className={passwordVisibilityToggleClasses}
-        disabled={disabled}
-        onClick={handleTogglePasswordVisibility}>
-        {!disabled && (
-          <span className={`${prefix}--assistive-text`}>
-            {passwordIsVisible ? hidePasswordLabel : showPasswordLabel}
-          </span>
-        )}
-        {passwordVisibilityIcon}
-      </button>
+      <Tooltip
+        align={align}
+        className={`${prefix}--toggle-password-tooltip`}
+        label={passwordIsVisible ? hidePasswordLabel : showPasswordLabel}>
+        <button
+          type="button"
+          className={passwordVisibilityToggleClasses}
```

Comment:
Does this variable need to change at all? Are the classes defined in it all still needed? If not, are there some associated styles we can remove in `_text-input.scss`?

Reply:
These styles were already there, however I tried to check the classes and found that we are adding/removing few classes based on disabled and tooltip alignments, although few of them we can remove I think so, will remove accordingly.